### ISSUE TITLE
Some key bindings don't work in some specific cases

### ISSUE THREAD

**Issue by bermudi** [romkatv/zsh4humans#143]
If `start-tmux` is set to `no` in `.zshrc` and a new shell is initialized by using either `su` or `su -`, completion commands are not recognized. Using `tab` key or `Ctrl` + `r` does absolutely nothing. Other key bindings like `Ctrl` + `a` || `w` || `e` work as expected.

If the the shell is either initialized normally (TTY or desktop login for example) then `tab` key and `Ctrl` + `r` key binding work as expected even if `start-tmux` is set to `no`.

**Comment by romkatv** [comment]
Thanks for the instructions. I've reproduced this. Will try to fix this later.

May I ask why you set `start-tmux` to `no`? Widgets based on `fzf` feel quite bad in this mode because they constantly jerk prompt back and forth. If integrated tmux is causing issues to you, I'd like to fix them.

**Comment by bermudi** [comment]
I was just testing things out and I guess I pushed that change to my dotfiles repo without realizing and I immediately noticed once I cloned. The one issue I'm having with system tmux is that p10k conf doesn't give me all prompt style options and integrated does. I think I just need to check [this](https://github.com/romkatv/powerlevel10k#some-prompt-styles-are-missing-from-the-configuration-wizard) but I've been doing other things.

Thanks for all your great effort btw, I changed from omz and wow.
